x-pack/plugins/observability_solution/synthetics/server/common/inline_to_zip.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { Logger } from '@kbn/logging';
+import archiver from 'archiver';
+import { MemWritable } from './mem_writable';
+
+function wrapInlineInProject(inlineJourney: string) {
+  return `import { journey, step, expect } from '@elastic/synthetics';
```

Comment:
We need to add the recently added `mfa` to this list. To make the Inline monitors work - https://github.com/elastic/synthetics/pull/957

Reply:
We will need to always be very careful about breaking this in the future.

Thanks for pointing this out!

[32bf01a](https://github.com/elastic/kibana/pull/176094/commits/32bf01a602eade87786bf052dd5aa2193536bda2)